src/mod_manager.cpp

```diff
@@ -18,6 +18,16 @@
 #define MOD_SEARCH_FILE "modinfo.json"
 
 static std::unordered_set<std::string> obsolete_mod_list;
+static const std::vector<std::pair<std::string, std::string> > categories = {
+    {"weapons", _("WEAPON MODS")},
```

Comment:
I suspect this wont translate correctly. The static vector is initialized before `main` is called and therefor before the translation system is set up. You can circumvent this by moving it into a function:
```C++
const vector &get_categories() {
    static const vector categories = ...;
    return categories;
}
```

The vector in the function is initialized when the function is called the first time, which should be after the translation system has been initialized.